src/subscription.ts

```diff
@@ -417,7 +446,15 @@ export class Subscription extends EventEmitter {
           reqOpts,
           gaxOpts,
         },
-        callback);
+
+
+        (err: Error, resp: google.protobuf.Empty) => {
```

Comment:
This feels new.  Why the behavior change?

Reply:
Oops, the implementation was done for before merge repository, i think i missed that while master merge.